site/overview/faqs.md

```diff
@@ -0,0 +1,154 @@
+
+::: details Answer
+This can happen when UserOperations become underpriced, frequently due to fee market movement between when gas and fees are estimations and when the user operation is actually submitted.
+
+You may experience this when calling the [`waitForUserOperationTransaction`](/packages/aa-core/provider/waitForUserOperationTransaction.html#waitForUserOperationTransaction) method. It may throw an error if it does not find the mined user operation within it’s retry limits. You can mitigate this by defining a more flexible retry period when constructing a [provider](/packages/aa-core/provider/constructor.html#constructor) (i.e. `txMaxRetries`, `txRetryIntervalMs`, `txRetryMultiplier` in `opts`). If your user operation continues to be delayed beyond a limit you are willing to wait, you can resubmit the user operation using [`dropAndReplaceUserOperation`](/packages/aa-core/provider/dropAndReplaceUserOperation.html#dropandreplaceuseroperation).
+:::
+
+### Are User Operations protected from MEV bots?
+
+::: details Answer
+Right now, user operations are sent to a private mempool for all networks other than Polygon, where there is no way to do this. We are actively involved in proposals for a peer-to-peer mempool standard.
+:::
+
+### Can I simulate user operations the same way I simulate transactions?
+
+::: details Answer
+Yes! Check out [this guide](/tutorials/sim-user-operation.html).
+:::
+
+## Gas Estimation
+
+### How does gas estimation for 4337 smart contract accounts work?
+
+::: details Answer
+Our bundler estimates gas and submits User Operations under the hood of the aa-sdk. Our gas estimations are just that, estimations that optimize for user operations landing on chain, and you may need to adjust gas limits based on your needs using [overrides](/packages/aa-core/provider/types/userOperationOverrides.html). Learn more about gas estimation and how it is implemented in our [bundler](https://www.alchemy.com/blog/erc-4337-gas-estimation).
+
+There are many nuances and edge cases that our bundler considers especially for L2’s. Learn more [here](https://www.alchemy.com/blog/l2-gas-and-signature-aggregators).
+
+We recommend adding error handling when sending a `UserOperation` to handle potential gas and fee changes due to market movement. Learn more about [frequent errors](#common-errors).
+:::
+
+## Gas Manager
+
+### What tiers support gas sponsorship?
+
+::: details Answer
+Gas sponsorship is available on testnet for all tiers. For support on mainnet, you must be on a paying tier (i.e. Growth tier and above). Learn more about our different pricing tiers [here](https://docs.alchemy.com/reference/gas-manager-coverage-api-pricing#fee-logic).
+:::
+
+### How is gas sponsored? Do I need to fund the gas manager?
+
+::: details Answer
+We front the gas for your application and put the USD equivalent on your bill at the end of the month. No need to worry about pre-funding the gas manager or conversions, we’ve got you covered!
```

Comment:
We'd want this to answer the first question too right? 
> How is gas sponsored?

The answer to this should be tied to paymasters generally, and then lead into how we handle it with our Gas Manager. Feel free to push back here.

Reply:
What other details do you think would be useful here? I actually think this answers the question as this isn't supposed to be a guide for how to sponsor. I can link that guide at the end.